src/Sentry/Internal/DebugStackTrace.cs

```diff
@@ -358,6 +358,25 @@ internal static SentryStackFrame ParseNativeAOTToString(string info)
         return frame;
     }
 
+    internal static string GetRepositoryRelativePath(Assembly moduleAssembly, string frameFileName,
+        SentryOptions options)
+    {
+        if (AttributeReader.TryGetRepositoryRoot(moduleAssembly) is not { } repoRoot ||
+            !frameFileName.StartsWith(repoRoot))
```

Comment:
This might be problematic if the code was built on a Windows machine (with `\` path separators) and run on a *nix variant (with `/` path separators). Let's make this more robust.